util/dvsim/SimCfg.py

```diff
@@ -72,6 +72,10 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
         self.en_build_modes = args.build_modes.copy()
         self.run_opts = []
         self.run_opts.extend(args.run_opts)
+        if args.build_timeout_mins:
+            self.build_timeout_mins = args.build_timeout_mins
+        if args.run_timeout_mins:
+            self.run_timeout_mins = args.run_timeout_mins
```

Comment:
This is not needed - we make `args` as an instance member of `*Cfg` object. So in `Deploy`, you can reference `self.sim_cfg.args.*_timeout_mins` directly.

Reply:
May be a bad trade-off: this does it once, while in the deploy object it would have to be done once per object. By the way, is there a function or piece of code that deals with the priority of dvsim args vs cfg/mode settings?

Reply:
Not really - in fact we are reducing memory by not allocating `*_timeout_mins` fields in `SimCfg` - `Deploy` derivatives will directly reference it using `self.sim_cfg.args`.

Reply:
Reducing memory for a config is peanuts vs per deploy object overhead. clkmgr config runs close to a thousand runs for example.